pkg/runtime/wfengine/activity.go

```diff
@@ -117,6 +120,11 @@ func (a *activityActor) InvokeReminder(ctx context.Context, actorID string, remi
 		if errors.Is(err, context.DeadlineExceeded) {
```

Comment:
Nit: please convert these `if..else if..else if..else if..else` to a switch for readability

Reply:
Novice Go question: how can I convert `else if _, ok := err.(recoverableError); ok` into a valid `case` statement?

Reply:
Use errors.As: https://pkg.go.dev/errors#As

Reply:
Thanks - Done